Allow skipping non-finite values in float series (#85)
neptune-ai/neptune-client-scale: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

> ## [0.7.3] - pending

Reviewer: `0.7.3` is already released

### Added

> - Neptune will now skip non-finite metric values by default, instead of raising an error. This can be configured using
>   the `NEPTUNE_SKIP_NON_FINITE_METRICS` environment variable ([#85](https://github.com/neptune-ai/neptune-client-scale/pull/85))

Reviewer: ```suggestion
- Neptune skips non-finite metric values by default, instead of raising an error. You can configure this behavior with
  the `NEPTUNE_SKIP_NON_FINITE_METRICS` environment variable ([#85](https://github.com/neptune-ai/neptune-client-scale/pull/85))
```
